v19.2/start-a-local-cluster-in-docker-windows.md

```diff
@@ -26,171 +26,179 @@ Running multiple nodes on a single host is useful for testing CockroachDB, but i
+   {{site.data.alerts.callout_info}}Be sure to replace <code>&#60;username&#62;</code> in the <code>-v</code> flag with your actual username.{{site.data.alerts.end}}
 
-    ~~~ powershell
-    PS C:\Users\username> docker run -d `
-    --name=roach1 `
-    --hostname=roach1 `
-    --net=roachnet `
-    -p 26257:26257 -p 8080:8080  `
-    -v "//c/Users/<username>/cockroach-data/roach1:/cockroach/cockroach-data"  `
-    {{page.release_info.docker_image}}:{{page.release_info.version}} start `
-    --insecure `
-    --join=roach1,roach2,roach3
-    ~~~
+   ```powershell
+   PS C:\Users\username> docker run -d `
+   --name=roach1 `
+   --hostname=roach1 `
+   --net=roachnet `
+   -p 26257:26257 -p 8080:8080  `
+   -v "//c/Users/<username>/cockroach-data/roach1:/cockroach/cockroach-data"  `
+   {{page.release_info.docker_image}}:{{page.release_info.version}} start `
+   --insecure `
+   --join=roach1,roach2,roach3
+   ```
 
 2. This command creates a container and starts the first CockroachDB node inside it. Take a moment to understand each part:
-    - `docker run`: The Docker command to start a new container.
-    - `-d`: This flag runs the container in the background so you can continue the next steps in the same shell.
-    - `--name`: The name for the container. This is optional, but a custom name makes it significantly easier to reference the container in other commands, for example, when opening a Bash session in the container or stopping the container.
-    - `--hostname`: The hostname for the container. You will use this to join other containers/nodes to the cluster.
-    - `--net`: The bridge network for the container to join. See step 1 for more details.
-    - `-p 26257:26257 -p 8080:8080`: These flags map the default port for inter-node and client-node communication (`26257`) and the default port for HTTP requests to the Admin UI (`8080`) from the container to the host. This enables inter-container communication and makes it possible to call up the Admin UI from a browser.
-    - `-v "//c/Users/<username>/cockroach-data/roach1:/cockroach/cockroach-data"`: This flag mounts a host directory as a data volume. This means that data and logs for this node will be stored in `Users/<username>/cockroach-data/roach1` on the host and will persist after the container is stopped or deleted. For more details, see Docker's <a href="https://docs.docker.com/engine/admin/volumes/bind-mounts/">Bind Mounts</a> topic.
-    - `{{page.release_info.docker_image}}:{{page.release_info.version}} start --insecure --join`: The CockroachDB command to [start a node](cockroach-start.html) in the container in insecure mode. The `--join` flag specifies the `hostname` of each node that will initially comprise your cluster. Otherwise, all [`cockroach start`](cockroach-start.html) defaults are accepted. Note that since each node is in a unique container, using identical default ports won’t cause conflicts.
+
```

Comment:
This is an unnecessary formatting change that breaks the steps numbering. Please revert the change.